tests/playwright/sanity/modules/nested-accordion/nested-accordion.test.ts

```diff
@@ -243,4 +244,24 @@ test.describe( 'Nested Accordion experiment is active @nested-accordion', () =>
 
 		expect( nestedAccordionWidth ).toEqual( containerWidth );
 	} );
+
+	test( 'Nested Accordion with inner Nested Accordion', async ( { browser }, testInfo ) => {
+		// Act
+		const page = await browser.newPage(),
+			wpAdmin = new WpAdminPage( page, testInfo ),
+			editor = await wpAdmin.openNewPage(),
+			frame = editor.getPreviewFrame();
+
+		await test.step( 'Load Template', async () => {
+			const filePath = _path.resolve( __dirname, `./templates/nested-accordion-inner.json` );
+			await editor.loadTemplate( filePath, false );
+			await frame.waitForSelector( '.elementor-widget-n-accordion' );
+			await editor.closeNavigatorIfOpen();
+		} );
+
+		await test.step( 'Verify that the inner accordion doesn\'t inherit styling from parent', async () => {
+			await expect.soft( await frame.locator( '.e-n-accordion-item-title' ).first() ).toHaveCSS( 'background-color', 'rgb(110, 193, 228)' );
+			await expect.soft( await frame.locator( '.e-n-accordion-item > .e-con .e-n-accordion-item-title' ).first() ).toHaveCSS( 'background-color', 'rgba(0, 0, 0, 0)' );
```

Comment:
Checking the only background color is enough?

Reply:
I will do a screenshot test here.